File: Domain/Classification.cs
using ScotlandsMountains.Domain.Values;

namespace ScotlandsMountains.Domain;

public class Classification : Entity
{
    public string SingularName { get; set; }

    public int DisplayOrder { get; set; }

    public string Description { get; set; }

    public List<MountainSummary> Mountains { get; set; }
}

File: UI/src/app/models/classification.ts
export interface Classification {
    id: string;
    name: string;
    singularName: string;
    description: string;
    mountainsCount: number;
    mountains: MountainSummary[];
}

export interface MountainSummary {
    id: string;
    name: string;
    height: number;
    Longitude: number;
    Latitude: number;
}
